.gitignore

```diff
@@ -1,37 +1,223 @@
+
```

Comment:
The .gitignore should only contains things relevant to the project. Other things such as files specifically for your editor should be in your personal global gitignore.